AK/kmalloc.h

```diff
@@ -1,13 +1,27 @@
 /*
  * Copyright (c) 2018-2020, Andreas Kling <kling@serenityos.org>
+ * Copyright (c) 2021, Daniel Bertalan <dani@danielbertalan.dev>
  *
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
 #pragma once
 
-#ifndef __serenity__
+#if defined(KERNEL)
+#    include <Kernel/Heap/kmalloc.h>
+#else
 #    include <new>
+#    include <stdlib.h>
+
+#    define kcalloc calloc
+#    define kmalloc malloc
+#    define kmalloc_good_size malloc_good_size
+#    define kfree free
+#    define krealloc realloc
+#endif
+
+#ifndef __serenity__
+#    include <AK/Types.h>
```

Comment:
Is this indentation commonplace with nested defines?

Reply:
It's like everywhere in our codebase.

IMO, it's still more readable than the other option, which is no indentation.